src/Base/BaseClass.cpp

```diff
@@ -23,11 +23,6 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_

```

Comment:
assert() is used in several functions of BaseClass

Reply:
re-added